ReactiveObjC/UIControl+RACSignalSupportPrivate.h

```diff
@@ -27,3 +29,5 @@
 - (RACChannelTerminal *)rac_channelForControlEvents:(UIControlEvents)controlEvents key:(NSString *)key nilValue:(id)nilValue;
```

Comment:
Isn't `nilValue` nullable? e.g. `- (RACChannelTerminal *)rac_newSelectedSegmentIndexChannelWithNilValue:(nullable NSNumber *)nilValue;` calls through to this with a nullable value